bundles/at.medevit.elexis.text.docx/rsc/script/win/doc2sumatraprint.ps1

```diff
@@ -0,0 +1,27 @@
+param(
+[string]$doc_path,
+[string]$printer,
+[string]$tray)

```

Comment:
Bzgl. `SumatraPDF.exe` - Ist das gut das SumatraPDF.exe file mitzuführen, bzw. als binary zu halten?  Unter was für einer Lizenz steht das, sollten wir nicht enstsprechend info dazu führen?

Reply:
Okay. Das war halt auch einfach dabei. Werde das abklären und in separatem commit machen. Weiss auch gar nicht woher das kommt und ob die Version aktuell ist.

Reply:
Diese Exe wurde von Thomas am 25.2.2019 zu medelexis-3 hinzugeführt. S.a.
https://redmine.medelexis.ch/issues/14911
https://redmine.medelexis.ch/issues/14885

Reply:
Das Exe ist 32-Bit. Es gibt auch deutlich neuere Versionen und eine 64-bit Version.
Switch auf 64-bit und Versions Upgrade mache ich in einem separaten Ticket.